### DISCUSSION TITLE
using pandoc, tilde fenced code blocks not working?

### DISCUSSION THREAD

**Discussion by uvtc** [jgm/djot#333] [General]
Using pandoc (v3.6.1) to convert a small djot file to html, the tilde-delimited codeblock syntax isn't working, but backtick delimited is.

~~~~~~
~~~
this not getting rendered
as a code block
~~~

```
but this is
```
~~~~~~

My understanding is that tilde-delimited code blocks are supported in djot, and it works fine in the [live sandbox](https://djot.net/playground/). Any idea why it's not working with pandoc?

**Comment by jgm** [comment]
The tilde form isn't actually documented:
https://htmlpreview.github.io/?https://github.com/jgm/djot/blob/master/doc/syntax.html#code-block
(I don't remember all the history about this, but it may be that it was supported in earlier versions and then removed to conform to Principle 11 in the djot README.)

**Reply by jgm** [reply]
Ref: #41

**Reply by uvtc** [reply]
Thanks, John. I just added what I hope is a useful comment to that issue.
